TYPO3.Media/Resources/Private/Partials/FlashMessages.html

```diff
@@ -1,22 +1,10 @@
+{namespace neos=TYPO3\Neos\ViewHelpers}
 <f:flashMessages as="flashMessages">
-	<f:for each="{flashMessages}" as="flashMessage">
-		<f:if condition="{flashMessage.severity} == 'OK'">
-			<div class="neos-alert neos-alert-success">
-		</f:if>
-		<f:if condition="{flashMessage.severity} == 'Notice'">
-			<div class="neos-alert neos-alert-notice">
-		</f:if>
-		<f:if condition="{flashMessage.severity} == 'Warning'">
-			<div class="neos-alert neos-alert-warning">
-		</f:if>
-		<f:if condition="{flashMessage.severity} == 'Error'">
-			<div class="neos-alert neos-alert-error">
-		</f:if>
-		<a class="neos-close" data-dismiss="alert" href="#">×</a>
-		<f:if condition="{flashMessage.title}">
-			<h4 class="neos-alert-heading">{flashMessage.title}</h4>
-		</f:if>
-		{flashMessage}
-		</div>
-	</f:for>
-</f:flashMessages>
+	<f:if condition="{flashMessages -> f:count()} > 0">
+		<ul id="neos-notifications-inline">
+			<f:for each="{flashMessages}" as="flashMessage">
+				<li data-type="{flashMessage.severity -> f:format.case(mode: 'lower')}">{neos:backend.translate(id: 'flashMessage.{flashMessage.code}', arguments: flashMessage.arguments, value: flashMessage)}</li>
+			</f:for>
+		</ul>
+	</f:if>
+</f:flashMessages>
```

Comment:
technically this output is wrong and belongs to Neos only, but since we are dropping the standalone media templates it's okay for now